src/components/dashboard/main-content/course-enrollments/course-cards/mark-complete-modal/tests/MarkCompleteModal.test.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/jsx-no-constructed-context-values */
```

Comment:
[question] Is this ESLint rule disabling the need to use `useMemo` with the context values? Curious why you're ignoring it in this test file but not in others. We should be consistent about how we handle this ESLint rule.